util/travis/linux-cibuild.sh

```diff
@@ -18,6 +18,11 @@ if [ "$(echo ${CXX} | cut -c1-5)" = "clang" ]; then
   export CXXFLAGS="${CXXFLAGS} -Qunused-arguments -DSEQAN_IGNORE_MISSING_OPENMP=1"
 fi
 
+# compile with c++-17 if g++-7 is used.
+if [ "$(echo ${CXX})" = "g++-7" ]; then
```

Comment:
that means g++-7 will ALWAYS build with -std=c++17? I would want that g++-7 also tests c++14

Reply:
Sure, but adding another build to Travis will make it more often reduce the parallel builds to sequential execution and wait times will be to long. I am preparing in parallel ci for Unix on Jenkins. As soon we have this we can revert this build here

Reply:
true

Reply:
In the nightlies we have g++7 with std=c++14, too, but for travis it's ok to only test one combination from my POV. (as long as c++17 is checked at all)